app/index.js

```diff
@@ -183,16 +183,17 @@ app.on('ready', () => {
       cb(true)
       return
     }
+
     // Tell the page to show an unlocked icon. Note this is sent to the main
     // window webcontents, not the webview webcontents
-    BrowserWindow.getAllWindows().map((win) => {
-      win.webContents.send(messages.CERT_ERROR, {
-        url,
-        error,
-        cert
-      })
+    webContents.hostWebContents.send(messages.CERT_ERROR, {
```

Comment:
this throws `cannot read property 'send' of null` on startup when the restored page has an invalid cert - need to check that hostWebContents exists